web/html/src/components/table/Table.js

```diff
@@ -0,0 +1,94 @@
+// @flow
+import * as React from 'react';
+import { TableDataHandler } from './TableDataHandler';
+import { SearchField } from './SearchField';
+import { Column } from './Column';
+
+type TableProps = {
```

Comment:
Just asking: do we really need to duplicate all this long `Props` list here in `TableProps` and in `TableDataHandler Props`?

Reply:
yes, otherwise the storybook docs addon doesn't list those properties